keyboards/handwired/tritium_numpad/keymaps/blu/keymap.c

```diff
@@ -0,0 +1,24 @@
+#include QMK_KEYBOARD_H
+
+void keyboard_pre_init_user(void)
+{
+    setPinOutput(B0);
+}
+
+uint32_t layer_state_set_user(uint32_t state)
+{
+    switch (biton32(state)) {
+    case 0:
+        writePinHigh(B0);
+        break;
+    case 1:
+        writePinLow(B0);
+        break;
+	}
+    return state;
+}
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+	[0] = LAYOUT_ortho_6x4(KC_NLCK, KC_PSLS, KC_PAST, KC_PMNS, KC_P7, KC_P8, KC_P9, KC_PPLS, KC_P4, KC_P5, KC_P6, KC_BSPC, KC_P1, KC_P2, KC_P3, KC_PENT, KC_P0, KC_UP, KC_PDOT, TT(1), KC_LEFT, KC_DOWN, KC_RGHT, BL_STEP),
```

Comment:
You might want to format this so it's easier to read.

Reply:
It came from the QMK keymap generator.  I'll reformat it.